ballerina-tests/tests/resources/documents/interfaces_implementing_interface_introsepction.graphql

```diff
@@ -0,0 +1,35 @@
+query {
+  # interfaces
+  Animalia: __type(name: "Animalia") {
+    ...details
+  }
+  Equine: __type(name: "Equine") {
+    ...details
+  }
+  # objects
+  Horse: __type(name: "Horse") {
+    ...details
+  }
+  Donkey: __type(name: "Donkey") {
+    ...details
+  }
+  Mule: __type(name: "Mule") {
+    ...details
+  }
+  Dog: __type(name: "Dog") {
+    ...details
+  }
+}
```

Comment:
This is okay. But I'd instead look into all the types since this does not clearly show how the interfaces implementing interfaces show in the schema. For instance, we need to know there is a type where its kind is `Interface`, it has fields, and its `interfaces` field has values.

Reply:
Introduced a fragment to query interfaces